styles/styles.css

```diff
@@ -0,0 +1,337 @@
+@import url('https://fonts.googleapis.com/css?family=Archivo+Black|Merriweather:400,400i');
+
+.flex-container {
+    display: flex;
+    /*Need to decide on columns, rows, or other approach*/
+    /*May also need to figure out if you can do multiple grids in one page and if so whether that is best approach here*/
+}
+
+a:link {
+  text-decoration: none;
+}
+  
+a:visited {
+  text-decoration: none;
+}
+  
+#banner {
+  background-color: #093b60;
+  height: 18em;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+}
+
+a.subtitle:visited {
+  color: #093b60;
+}
+
+#letterhead {
+  display: flex;
+  flex-direction: column;
+  
+  margin-top: 3%;
+  color: #81B642;
+  font-family: Merriweather, Times, serif;
+  font-style: italic;
+  text-align: center;
+  font-size: 150%;
+}
+
+.filler {
+  width: 0%;
+  
+ 
+}
+
+#toplogo {
+  max-width: 100%;
+  height: auto;
+}
+
+#cart {
+  display: flex;
+  flex-direction: column;
+  justify-content: right;
+  margin-top: 2rem;
+   
+}
+
+.top-cart {
+  color: white;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+}
+
+
+#carticon {
+/*Size to make small; I've just shrunk it a bit for now*/
+width: 1.5em;
+
+}
+
+.checkout-button {
+  /*Green background, position beneath cart icon */
+    background-color: #81B642;
+    height: 30px;
+    width: 140px;
+    color: white;
+    font-family: "Archivo Black", Geneva, sans-serif;
+    font-weight: bold;
+    font-size: 75%;
+    line-height: 30px;
+    text-align: center;
+    margin: 10px;
+}  
+
+h1 {
+  
+/*Size */
+}
+
+#subbanner {
+  background-color: #ebebeb;
+  height: 4em;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.subtitle {
+ /*Position away from left margin and check font */
+  color: #093b60;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+  margin-left: 10rem;
+}
+
+#welcome {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 1rem;
+}
+
+#hugpicture {
+  width: 85%;
+}
+
+h2 {
+  font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
+  color: #093b60;
+  font-size: 1.75em;
+  font-weight: 900;
+}
+
+.lead-in {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  color: #3C3C3C;
+  width: 30rem;
+
+}
+
+.profiles {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  justify-content: center;
+}
+
+.profile-card {
+  width: 200px;
```

Comment:
You'll want to use a percent for this. You can constrain how small or large it gets with max-width and min-width

Reply:
I'm wondering if you could offer some meta-advice on this one. My initial feeling is that hardcoding my profile-cards is the way to go. I'd like to try building an alternate version to test out your recommendation, though. What's the best way to use document control to accomplish this?

I was going to create a new file ("index-alt.html") to build the profile-cards using percentage. Of course, I'll probably be making other tweaks to index.html as things go along and the two files could get out of sync in other ways. Is there a best practice to organize them?

Thanks!

Reply:
For now, the easiest method will be to just create different html files.

If you want to do it right, you can create a different branch from master and do your experimental work on that. We haven't learned about branching on git, but you're welcome to investigate it. If you do create a new branch, you'll need to do another PR in order for feedback. You'd be able to keep your master PR up, and also do a PR against your new branch. It depends on where you want to spend your time learning.